docs/conf.py

```diff
@@ -247,22 +236,14 @@ def patched_parse(self):
 # Usually you set "language" from the command line for these cases.
 language = None
 
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This pattern also affects html_static_path and html_extra_path .
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
-
-# The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
-
 
 # -- Options for HTML output -------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
-html_style = 'css/override_theme.css'
+# html_theme = 'sphinx_rtd_theme'
```

Comment:
Are the commented out HTML commands still needed now that the MDO RTD theme is being used?